SharpPcap/LibPcap/PcapDevice.cs

```diff
@@ -267,15 +258,8 @@ public virtual System.Net.NetworkInformation.PhysicalAddress MacAddress
         {
             get
             {
-                ThrowIfNotOpen("device not open");
```

Comment:
this check not necessary anymore?

Reply:
Was not necessary to begin with, it's only using `PcapInterface.Addresses` and that does not require the device to be open.